src/controllers/userController.js

```diff
@@ -0,0 +1,81 @@
+const jwt = require("jsonwebtoken")
+const { checkEmptyValues, generateToken, throwError } = require("../utils");
+const { userServicve } = require("../services");
+const { detailUpdateUser } = userServicve;
+
+const detailSignUp = async (req, res) => {
+
+  const accessToken = req.headers.authorization;
+  const { id } = jwt.verify(accessToken, process.env.SECRET);
```

Comment:
```suggestion

const updateUserInfo = async (req, res) => {

  const accessToken = req.headers.authorization;
  const { id } = jwt.verify(accessToken, process.env.SECRET);
```

함수의 기능을 보니 사용자의 정보를 업데이트하는 것 같은데, 맞을까요? 함수명 명확히 해주시면 좋을 것 같습니다 :)